contracts/callcenter/src/lib.rs

```diff
@@ -79,6 +80,18 @@ impl Callcenter {
             false => Ok(caller == self_id),
         }
     }
+
+    /// Calls the `spend` function of the `contract` with no arguments, and the
+    /// given `points_limit`, assuming the function does not return. It will
+    /// then return that call's result.
+    pub fn call_spend_with_limit(
```

Comment:
So this is spending all gas-points up until the given `points_limit`? What I don't get is what the sentence "..., assuming the function does not return." -> which function, `spend`? `call_spend_with_limit`
"It will then return that call's result" -> which call's? And what is the return?

Reply:
I suppose I meant a `void/()` return on the `spend` call. I'll update to reflect this better.